pkg/compute/management_client.go

```diff
@@ -124,36 +127,62 @@ func (m *ManagementClient) updateResources(ctx context.Context) {
 		Resources: resources,
 	})
 	if err != nil {
-		log.Ctx(ctx).Error().Err(err).Msg("failed to send resource update to requester node")
```

Comment:
While I am thinking about this. Above on line 103 where we call `getNodeInfo` we do:
```go
nodeInfo := m.getNodeInfo(ctx)
	response, err := m.managementProxy.UpdateInfo(ctx, requests.UpdateInfoRequest{
		Info: nodeInfo,
	})
	if err != nil {
		log.Ctx(ctx).Error().Err(err).Msg("failed to send update info to requester node")
	}
// TODO response will be nil when err != nil and this code will panic (has panicked for me)
	if response.Accepted {
		log.Ctx(ctx).Debug().Msg("update info accepted")
	} else {
		log.Ctx(ctx).Error().Msgf("update info rejected: %s", response.Reason)
	}
```
Can we address the panic here?

Reply:
I think I intended this to return after logging the error.  Fixed in a96c1b00137a442f20a1ae8b920c9b632c6f7daa